### ISSUE TITLE
bug "/api/v0/version" 405 (Method Not Allowed)

### ISSUE THREAD

**Issue by parajbs** [RangerMauve/auto-js-ipfs#8]
Hello **@RangerMauve,**

I tested "RangerMauve/auto-js-ipfs" and it works partially.
I think I found a bug, here the error in Chrome and also in Brave:
```
GET http://localhost:45005/api/v0/version 405 (Method Not Allowed)
GET http://localhost:5001/api/v0/version 405 (Method Not Allowed)
```

I did a little research and found out that it needs to be a POST request sent to "/api/v0/version" via "IPFS API port" and not a GET request.

The way I see it, the GET request only works over the "IPFS Gateway port" and not over the "IPFS API port".

Second error:
I don't understand this error, but it also occurs in all browsers.
Here error:
```
Couldn't find readonly input
loadOptions @ example.js:73
```

@RangerMauve could she help and correct?

I've forked it myself, and am currently including "NFT.Storage, Pinata, Infura, filebase.com and more".
I am also trying to include directory upload.

**Comment by RangerMauve** [comment]
The `GET http://localhost:45005/api/v0/version 405 (Method Not Allowed)` API call erroring out is okay since it's just trying to detect whether there's something listening on that API endpoint in the first place.

Would you be able to produce a code snippet that reproduces the issue with `Couldn't find readonly input`?

**Comment by parajbs** [comment]
hello @RangerMauve and @ikreymer,

Sorry can't help at the moment, had to reinstall my computer and haven't gotten to reinstalling ipfs yet so I can test.

Find the project "RangerMauve/auto-js-ipfs" super great and had already integrated it into my private ipfs portal with tus-server.
I had already integrated Filebase and Storj.

Once I've got everything up and running again, I'll be happy to continue testing here.
I will definitely continue to follow this project.
